cmd/mesh/shared.go

```diff
@@ -18,15 +18,33 @@ package mesh
 
 import (
 	"fmt"
+	"io"
 	"os"
 
 	"istio.io/pkg/log"
 )
 
 const (
-	logFilePath = "./mesh-cli.log"
+	logFilePath = "./.mesh-cli.log"
 )
 
+func configLogsOrExit(args *rootArgs) {
+	_ = os.Remove(logFilePath)
```

Comment:
would this file be created back somewhere?not sure whether log.Configure does the work

Reply:
I've left a comment with some explanation. Normally it would be weird to have "config log" function have a side effect of deleting something :-)
I think a more accurate name is initLogsOrExit and in this case it's probably reasonable to have the delete in there, since this function is the first call in just about all commands.